### PR TITLE
chanValues changed to array, a testcase is fixed

### PR DESCRIPTION
**Pull request by AhmetCanSolak**
Please provide review, im considering to have another PR on other testcases too.

### CODE REVIEW

File: src/test/java/GS/test/GSBufferTests.java [hunk 1]
```diff
@@ -298,32 +285,26 @@ void GSBuffer_WriteChannel_range()
...
             buffertest.appendValue(1,1);
             buffertest.appendValue(1,3);
         } catch (Exception ex) {fail(ex);}
 
-        try {
-            buffertest.appendValue(1,2);
-        } catch(Exception ex) {System.out.println("active channel out of order pass");}
-
+        assertThrows(ActiveChanException.class, () -> buffertest.appendValue(1,2));
     }
```
Comment (bryantChhun) [inline src/test/java/GS/test/GSBufferTests.java:311]: add the "system.out.println('active channel out of order pass')"

File: src/test/java/GS/test/GSBufferTests.java [hunk 2]
```diff
@@ -107,13 +101,7 @@ void GSBuffer_VoltageToIntConversion_limit_high()
...
         } catch (Exception ex) {fail(ex);}
 
-        try {
-            buffertest.appendValue(10,1);
-        } catch(ActiveChanException ACex) {
-            fail(ACex);
-        } catch(VoltageRangeException VRex){
-            System.out.println("voltage limit high pass");
-        }
+        assertThrows(VoltageRangeException.class, () -> buffertest.appendValue(10,1));
```
Comment (bryantChhun) [inline src/test/java/GS/test/GSBufferTests.java:105]: and the "high pass" print here too

File: src/test/java/GS/test/GSBufferTests.java [hunk 3]
```diff
@@ -88,13 +88,7 @@ void GSBuffer_VoltageToIntConversion_limit_low()
...
 
-        try {
-            buffertest.appendValue(-10,1);
-        } catch(ActiveChanException ACex) {
-            fail(ACex);
-        } catch(VoltageRangeException VRex){
-            System.out.println("voltage limit low pass");
-        }
+        assertThrows(VoltageRangeException.class, () -> buffertest.appendValue(-10,1));
     }
```
Comment (bryantChhun) [inline src/test/java/GS/test/GSBufferTests.java:93]: can you add the "voltage limit low pass" statement here?

File: src/test/java/GS/test/GSBufferTests.java (hunk 1 above)
Comment (AhmetCanSolak) [reply src/test/java/GS/test/GSBufferTests.java:311]: added.

**Review by bryantChhun — CHANGES_REQUESTED** [review]
put some printlns back in

File: src/test/java/GS/test/GSBufferTests.java (hunk 3 above)
Comment (AhmetCanSolak) [reply src/test/java/GS/test/GSBufferTests.java:93]: done

File: src/test/java/GS/test/GSBufferTests.java (hunk 2 above)
Comment (AhmetCanSolak) [reply src/test/java/GS/test/GSBufferTests.java:105]: done

**Review by bryantChhun — APPROVED** [review]

**Pull request merged**